frontend/src/workbench/Site.tsx

```diff
@@ -11,128 +11,119 @@
  *     Obeo - initial API and implementation
  *******************************************************************************/
 
-import MuiAccordion from '@material-ui/core/Accordion';
-import AccordionDetails from '@material-ui/core/AccordionDetails';
-import MuiAccordionSummary from '@material-ui/core/AccordionSummary';
-import MuiCollapse, { CollapseProps } from '@material-ui/core/Collapse';
-import { makeStyles, withStyles } from '@material-ui/core/styles';
-import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import IconButton from '@material-ui/core/IconButton';
+import { makeStyles } from '@material-ui/core/styles';
+import Tooltip from '@material-ui/core/Tooltip';
+import Typography from '@material-ui/core/Typography';
 import React, { useState } from 'react';
 import { SiteProps } from './Site.types';
 
 const useSiteStyles = makeStyles((theme) => ({
-  site: {
-    display: 'flex',
-    flexDirection: 'column',
-  },
-  expanded: {
-    flexGrow: 1,
-    overflow: 'auto',
-  },
-  accordionDetailsRoot: {
-    display: 'block',
-    padding: '0px',
+  leftSite: {
+    display: 'grid',
+    gridTemplateColumns: '32px 1fr',
```

Comment:
I see that you are using several grids using only one dimension (here only one row), could you use the flexbox layout instead since we had some "hard to debug" performance issues with a large number of grids in the past.